sys/linux/filesystem.txt

```diff
@@ -899,6 +899,15 @@ nilfs2_options [
 	order_strict	stringnoz["order=strict"]
 ] [varlen]
 
+squashfs_options [
+	errors		fs_opt["errors", stringnoz[squashfs_error]]
+	threads_str	fs_opt["threads", stringnoz[squashfs_thread]]
+	threads_num	fs_opt_dec["threads", int32]
+] [varlen]
+
+squashfs_error = "continue", "panic"
```

Comment:
We need to avoid "panic" option b/c the fuzzer can provide invalid image, but if kernel panics, it's not a kernel bug. We remove "panic" in executor anyway.

Reply:
Removed the "panic" option